strings.md

```diff
@@ -1009,6 +1010,31 @@ With `~2,2f`:
 
 And we're happy with this result.
 
+### Iteration
+Create a string from a list with iteration construct `~{str~}`:
+~~~lisp
+(format t "~{~A, ~}" '(a b c))
+;; A, B, C,
+~~~
+
+using `~^` to avoid printing the comma and space after the last element:
+~~~lisp
+(format t "~{~A~^, ~}" '(a b c))
+;; A, B, C
+~~~
+
+`~:{str~}` is similar but for a list of sublists:
+~~~lisp
+(format nil "Pairs:~:{ <~S,~S>~} ." '((a 1) (b 2) (c 3)))
```

Comment:
I think using real words instead of two more symbols (< and >) would be more readable. As a newcomer we might not understand straight if < is part of the directive or not.

Reply:
changed!